tests/test_predictit.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+
+import pandas as pd
+
+import ergo
+
+
+def test_refresh(predictit):
+    predictit.refresh_markets()
+    assert "markets" in predictit._data
+
+
+def test_markets(predictit):
+    markets = list(predictit.markets)[0:10]
+    for market in markets:
+        assert type(market) is ergo.PredictItMarket
+        market_other = predictit.get_market(market.id)
+        assert market.name == market_other.name
+
+
+def test_market_attributes(predictit_markets):
+    attrs = {
+        "predictit": ergo.PredictIt,
+        "api_url": str,
+        "id": int,
+        "name": str,
+        "shortName": str,
+        "image": str,
+        "url": str,
+        "status": str,
+        "timestamp": datetime.datetime,
+    }
+    for market in predictit_markets:
+        for attr in attrs.items():
+            assert type(getattr(market, attr[0])) is attr[1]
+
+
+def test_market_questions(predictit_markets):
```

Comment:
I haven't looked carefully at this test and the tests below, but I'd guess that if I were to look at them more carefully, I'd feel confused about what they were testing.

I think to improve the tests, you should:
1. Delete some of them (if you want)
2. For each test, identify: what crucial functionality are we relying on the Predict API having? I think plausibly the tests that you'd want will look pretty different from what you have, such that you'll want to basically start over with writing the tests.
3. Explain what that functionality is on a docstring on the test
4. Make sure that the test directly tests that crucial functionality

Reply:
I'd just delete this test